src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacy.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Feature from 'app/components/acl/feature';
+import FeatureDisabled from 'app/components/acl/featureDisabled';
+import {PanelAlert} from 'app/components/panels';
+import {t} from 'app/locale';
+
+import ProjectDataPrivacyContent, {
+  ProjectDataPrivacyContentProps,
+} from './projectDataPrivacyContent';
+
+const ProjectDataPrivacy = ({params, organization}: ProjectDataPrivacyContentProps) => (
```

Comment:
Instead of importing props, you can access it through the component

```suggestion
const ProjectDataPrivacy = ({params, organization}: React.ComponentProps<ProjectDataPrivacyContent>) => (
```

Reply:
![image](https://user-images.githubusercontent.com/29228205/76067131-29889380-5f8f-11ea-9ea6-d5f723262f0f.png)